packages/web/package.json

```diff
@@ -15,12 +15,16 @@
   "main": "js/cjs/spirit-web.js",
   "module": "js/esm/spirit-web.js",
   "scripts": {
+    "icons:build": "shx mkdir -p public/icons && shx cp -r ../../node_modules/@lmc-eu/spirit-icons/dist/. public/icons",
```

Comment:
```sh
6:06:42 PM: $ shx mkdir -p public/icons && shx cp -r ../../node_modules/@lmc-eu/spirit-icons/dist/. public/icons
6:06:43 PM: cp: no such file or directory: ../../node_modules/@lmc-eu/spirit-icons/dist/
```

On Netlify, the base directory is set to `packages/web` for this deploy, the `./node_modules` are in this base, I think.

Reply:
Fixed. I checked the node_modules folder in Netlify and it was basically empty. So I used the dist folder in icons.